health_check/contrib/celery/apps.py

```diff
@@ -9,6 +12,11 @@ class HealthCheckConfig(AppConfig):
 
     def ready(self):
         from .backends import CeleryHealthCheck
+        if hasattr(settings, "HEALTHCHECK_CELERY_TIMEOUT"):
```

Comment:
I'd rather use `warnings.warn("foo bar", DeprecationWarning)` here over a warning log.

Reply:
Hadn't heard of this, very neat!